docs/source/manual/validation.rst

```diff
@@ -0,0 +1,254 @@
+.. _man-validation:
+
+#####################
+Dropwizard Validation
+#####################
+
+.. highlight:: text
+
+.. rubric:: Dropwizard comes with a host of validation tools out of the box to allow endpoints to return meaningful error messages when constraints are violated. Hibernate_ is packaged with Dropwizard, so what can be done in Hibernate, can be done with Dropwizard.
```

Comment:
Actually unber *Hibernate* name several projects are developed: ORM, Search, Validator, NoSQL mapper, etc...
I think, we should clarify that we mean Hibernate Validator.